src/chainlit/frontend/src/components/organisms/chat/inputBox/input.tsx

```diff
@@ -89,6 +92,21 @@ const Input = ({ onSubmit, onReply }: Props) => {
     }
   }, []);
 
+  const startAdornment = (
```

Comment:
I think this is not the correct place to display the icon button. Will update this comment with a suggestion